bin/rosmop

```diff
@@ -1,69 +1,9 @@
 #!/bin/bash
 
```

Comment:
The original script threw an error if the jar wasn't present, with a suggestion to call `mvn package`. Can we add that check here as well, since it's replacing the original?

Reply:
The `java` command will already give you an error for that case, no?

Reply:
Yeah, it's a class loader exception along with a stacktrace, which in my opinion is much less instructive.